examples/e-commerce/src/app/app.component.ts

```diff
@@ -11,4 +11,24 @@ export class AppComponent {
     indexName: 'instant_search',
     routing: true,
   };
+
+  onKeyUp = event => {
+    if (event.key !== 'Escape') {
+      return;
+    }
+    this.closeFilters();
+  };
+
+  public openFilters() {
+    document.body.classList.add('filtering');
+    window.scrollTo(0, 0);
```

Comment:
Comment about all InstantSearch flavours mobile demos:
Would it make sense to also scroll up when the pagination button is clicked?

Right now, it feels weird that you have to browse through the hits in the inverted order after switching to the next page. 

![2019-06-26 11 57 17](https://user-images.githubusercontent.com/2982512/60170841-98f79200-9809-11e9-8372-68472facec63.gif)